FXyz-Importers/src/main/java/org/fxyz3d/importers/obj/ObjImporter.java

```diff
@@ -105,7 +105,7 @@ public Model3D loadAsPoly(URL url) throws IOException {
 
     @Override
     public boolean isSupported(String extension) {
-        return SUPPORTED_EXT.equals(extension);
+        return SUPPORTED_EXT.equalsIgnoreCase(extension);
```

Comment:
Since you are at it, can you add the same to the Maya importer?

Reply:
Was planning to do that together with the Maya importer test in a separate PR